Gemfile.lock

```diff
@@ -110,10 +119,11 @@ DEPENDENCIES
   rake (~> 12.0)
   rb-fsevent
   rexml (~> 3.2.4)
-  rubocop (~> 0.38.0)
+  rubocop (~> 1.8)
+  rubocop-performance
   vcr
   webmock
   webrick (~> 1.7.0)
 
 BUNDLED WITH
-   1.17.3
+   2.2.3
```

Comment:
Latest is 2.2.5, is this version important?

Reply:
not very important, 2.x is compatible